packages/web-components/fast-foundation/src/radio/radio.ts

```diff
@@ -66,6 +53,12 @@ export class FASTRadio extends FormAssociatedRadio implements RadioControl {
     @observable
     public defaultSlottedNodes: Node[];
 
+    get radioGroup() {
```

Comment:
Let's make this private so it's not part of the public interface
```suggestion
    private get radioGroup() {
```

Reply:
Thanks, was just trying to debug what API extractor is upset about. Was close